pennylane_forest/numpy_wavefunction.py

```diff
@@ -90,25 +90,44 @@ def ev(self, A, wires):
           float: expectation value :math:`\left\langle{A}\right\rangle = \left\langle{\psi}\mid A\mid{\psi}\right\rangle`
         """
         # Expand the Hermitian observable over the entire subsystem
-        A = self.expand_one(A, wires)
+        A = self.expand(A, wires)
         return np.vdot(self.state, A @ self.state).real
 
-    def expand_one(self, U, wires):
```

Comment:
This method is an internal method, however --- it is not (or at least, there is no case where it should be) user or developer accessible, so a deprecation notice seems redundant. The new `expand` method should probably be renamed `_expand` instead

Reply:
Got it. Doesn't need to happen in this PR, but at some point, it'd be a good idea to add leading underscores to all functions that are really meant to be private, just to avoid confusion.